src/home/devrel-tracker.ts

```diff
@@ -18,8 +20,14 @@ export async function trackDevRelReferral(devGitHubId: number) {
   if (devRelCode && devRelCode != "done") {
     const url = `${WORKER_URL}/tracker?key=${encodeURIComponent(devGitHubId)}&value=${encodeURIComponent(devRelCode)}`;
```

Comment:
key is not needed anymore because the user ID is in the token. I think it's best to send the referral code and token in the body because it's a POST request